modules/fedimint-ln-common/src/endpoint_constants.rs

```diff
@@ -1,13 +1,29 @@
+pub const ADDRESS_ENDPOINT: &str = "/address";
 pub const ACCOUNT_ENDPOINT: &str = "account";
```

Comment:
Can we make a new file and separate these? Now its really difficult to tell which endpoints are for the gateway and which are for the federation

Reply:
Split the endpoints accordingly between `federation_endpoint_constants` and `gateway_endpoint_constants`